shared/chat/conversation/normal/index.native.tsx

```diff
@@ -110,16 +111,34 @@ const Conversation = React.memo(function Conversation(props: Props) {
     [dispatch, conversationIDKey]
   )
 
+  const insets = useSafeAreaInsets()
+  const headerHeight = 44
+  const height = Styles.dimensionHeight - insets.top - headerHeight
+
+  const safeStyle = React.useMemo(
+    () => ({
+      height,
+      maxHeight: height,
+      minHeight: height,
+      paddingBottom: insets.bottom,
+    }),
+    [height, insets.bottom]
+  )
+
   return (
-    <Kb.Box style={styles.innerContainer}>
-      <DropView style={styles.dropView} onDropped={onDropped}>
-        <Kb.Box2 direction="vertical" fullWidth={true} fullHeight={true}>
-          {props.threadLoadedOffline && <Offline />}
-          {innerComponent}
+    <Kb.Box2 direction="vertical" fullWidth={true} fullHeight={true} style={safeStyle}>
+      <Kb.KeyboardAvoidingView2 extraPadding={-insets.bottom}>
```

Comment:
this is the only screen that hides the tab bar so the handling is quite different. we just explicitly set it up